### PR TITLE
feat(Data Mapper): Github issue template

### PR DESCRIPTION
**Pull request by refortie**

### CODE REVIEW

**Review by hartra344 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/data_mapper.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  - refortie
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue.
+
+        This issue tracker is for reporting bugs found in Logic Apps UI (https://github.com/Azure/LogicAppsUX)
+        If you have a question about how to achieve something and are struggling with a broader Data Mapper issue, please post a question
+        inside of Logic App's Discussion repo: https://github.com/Azure/logicapps/discussions
```
Comment (ecfan) [inline .github/ISSUE_TEMPLATE/data_mapper.yml:14]: Grammar: `inside of the Azure Logic Apps' Discussion repo`

File: .github/ISSUE_TEMPLATE/data_mapper.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+labels: ['Data Mapper']
+assignees:
+  - refortie
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue.
+
+        This issue tracker is for reporting bugs found in Logic Apps UI (https://github.com/Azure/LogicAppsUX)
```
Comment (ecfan) [inline .github/ISSUE_TEMPLATE/data_mapper.yml:12]: `in Azure Logic Apps UI`

File: .github/ISSUE_TEMPLATE/data_mapper.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        If you have a question about how to achieve something and are struggling with a broader Data Mapper issue, please post a question
+        inside of Logic App's Discussion repo: https://github.com/Azure/logicapps/discussions
+        If it's an issue about the docs, please go to the Logic App's documentation repo (https://github.com/Azure/logicapps)
+
+        Before submitting a new bug/issue, please check the links below to see if there is a solution or question posted there already:
+         - Open Issue's tab: https://github.com/Azure/LogicAppsUX/issues?q=is%3Aissue+sort%3Aupdated-desc+position
```
Comment (ecfan) [inline .github/ISSUE_TEMPLATE/data_mapper.yml:18]: `Open Issues tab`

File: .github/ISSUE_TEMPLATE/data_mapper.yml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+         - Closed Issues tab: https://github.com/Azure/LogicAppsUX/issues?q=is%3Aissue+sort%3Aupdated-desc+position+is%3Aclosed
+
+        The more information you fill in, the better we can help you.
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the Bug with repro steps
```
Comment (ecfan) [inline .github/ISSUE_TEMPLATE/data_mapper.yml:26]: `Describe the bug with repro steps`

**Pull request merged**